community/assessments/projects/kubescape/self-assessment.md

```diff
@@ -0,0 +1,332 @@
+* IDE integration enabling users to find misconfigurations and vulnerabilities during development
+* CI/CD integrations enabling users to build security gating in their processes
+* Reachability analysis of vulnerabilities using eBPF
+* Validating Admission Policy library
+* Application security observability using eBPF
+* Network and seccomp policy generation based on workload behavior
+* Runtime attack detection based on predefined rules and anomalies
+
+### Actors
+
+#### Kubescape CLI
+
+![Kubescape CLI](cli-arch.png)
+
+Kubescape CLI is a simple tool written in Go, built on [OPA engine](https://github.com/open-policy-agent/opa) and [Grype](https://github.com/anchore/grype). It downloads frameworks and controls written in Rego and a vulnerability database. Using that input it looks for potential security and compliance issues in YAML/Helm files and vulnerabilities in container images.
+
+It can be used as an investigative tool for benchmarking Kubernetes security posture. In addition it can be planted in CI/CD processes for a security automation use case.
+
+#### Kubescape Operator
+
+![Kubescape Operator](operator-arch.png)
+
+Kubescape Operator gives multiple features that can be independently enabled. Its central design concept is to record scan results to a Kubernetes API object. The *Storage* component is a Kubernetes Aggregated API extension service, it stores the different results objects the Operator produces. The following main capabilities are available in the Operator:
+
+* Configuration scanning
+* Container image scanning and SBOM generation
+* Host scanning
+* Application security observability
+* Runtime detection of malicious activities
+
+##### Operator component
+
+This component is in charge of commanding and controlling the scans in the cluster. There are multiple configuration options for when and what to scan in the cluster. This component orchestrates these activities by triggering the Kubescape and KubeVuln components.
+
+##### Kubescape component
+
+This component is responsible for  configuration and host scanning. Like the CLI, it uses the [OPA engine](https://github.com/open-policy-agent/opa) to run the project's own Rego library of rules. It also scans the Kubernetes host to validate their configurations. The output of the scans is stored in the *Storage* component via the Kubernetes API and optionally sent to external API endpoints.
+
+##### KubeVuln component
+
+This component is responsible for image vulnerability scanning. It can either produce an SBOM object in the *Storage* and match the SBOM entries with vulnerabilities or rely on the *Node agent* to generate SBOM objects on the nodes and produce vulnerability manifests and VEX documents. All the results are stored in the *Storage* component via the Kubernetes API and optionally sent to external API endpoints.
+
+##### Node agent component
+
+This component has multiple purposes all bound to information available on Kubernetes nodes.
```

Comment:
I can't grammatically parse this.   Can you rephrase this?

Reply:
@JustinCappos does this increase clarity?

```suggestion
This component has multiple purposes which are all bound to information available on Kubernetes nodes.
```

Reply:
I don't understand what it means to have "a purpose bound to information"?   What is this trying to say?   Can you avoid the word bound and explain in more detail?

Reply:
> I don't understand what it means to have "a purpose bound to information"? What is this trying to say? Can you avoid the word bound and explain in more detail?

I wanted to emphasize here that the reason we have a node agent is because only through this sensor can we read and access the information available only on the Kubernetes nodes

Reply:
Okay, your explanation here helps.   Can you work something to this effect into the text so we can resolve the comment?